compose/config/config.py

```diff
@@ -505,6 +504,16 @@ def finalize_service(service_config):
     if 'restart' in service_dict:
         service_dict['restart'] = parse_restart_spec(service_dict['restart'])
 
+    if 'log_driver' in service_dict or 'log_opt' in service_dict:
+        if 'logging' not in service_dict:
+            service_dict['logging'] = {}
+        if 'log_driver' in service_dict:
+            service_dict['logging']['driver'] = service_dict['log_driver']
+            del service_dict['log_driver']
+        if 'log_opt' in service_dict:
+            service_dict['logging']['options'] = service_dict['log_opt']
+            del service_dict['log_opt']
```

Comment:
Could we put this into a new function called something like `normalize_v1_format()` (and call it from here)? That way when we go to remove V1, it's clear which normalize steps are done for legacy compatibility, and which ones are still required.